ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot

```diff
@@ -136,6 +136,43 @@ Verify Model Pods Are Deleted When No Inference Service Is Present
     [Teardown]   Clean Up Test Project    test_ns=no-infer-kserve
     ...    isvc_names=${models_names}   isvc_delete=${FALSE}
 
+Verify User Can Autoscale Using Concurrency
+    [Tags]    ODS-XXX    WatsonX
+    [Setup]    Set Project And Runtime    namespace=autoscale-con
+    ${flan_model_name}=    Set Variable    flan-t5-small-caikit
+    ${model_name}=    Create List    ${flan_model_name}
+    Compile Inference Service YAML    isvc_name=${flan_model_name}
+    ...    sa_name=${DEFAULT_BUCKET_SA_NAME}
+    ...    model_storage_uri=s3://ods-ci-wisdom/flan-t5-small/
+    ...    auto_scale=True
+    Deploy Model Via CLI    isvc_filepath=${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
+    ...    namespace=autoscale-con
+    Wait For Pods To Be Ready    label_selector=serving.kserve.io/inferenceservice=${flan_model_name}
+    ...    namespace=autoscale-con
+    ${host}=    Get KServe Inference Host Via CLI    isvc_name=${flan_model_name}   namespace=autoscale-con
+    ${body}=    Set Variable    '{"text": "At what temperature does liquid Nitrogen boil?"}'
+    ${header}=    Set Variable    'mm-model-id: ${flan_model_name}'
+    ${cmd} =     Set Variable     grpcurl -d
+    ${cmd} =    Catenate   ${cmd} ${body}
+    ${cmd} =    Catenate   ${cmd} -H ${header}
+    ${cmd} =    Catenate   ${cmd} -insecure
+    ${cmd} =    Catenate   ${cmd}  ${host}:443
+    ${cmd} =    Catenate   ${cmd}  "caikit.runtime.Nlp.NlpService/TextGenerationTaskPredict"
+    FOR    ${index}    IN RANGE    30
+            ${rc}  ${response}=   	Run And Return Rc And Output    ${cmd}&
+            Run Keyword And Continue On Failure  Should Be Equal As Integers     ${rc}  ${0}
+    END
```

Comment:
there is already a keyword to query and check response multiple times, could you re-use that one?

Reply:
nope, that keyword wait for response i am running the command in background so that it can almost run parllel. here is used "&"

Reply:
hmm did you try using it? was it preventing from scaling up?

In any case, I would avoid rewriting the entire keyword in the test case. Maybe we can still re-use one of the keyword and changing the behavior with a param. Wdyt?

Reply:
Indeed, I attempted it, but the outcome wasn't as anticipated during my local verification. As a result, I transitioned to this approach, which seems more appropriate in this context. The reason behind this shift is that when aiming for parallel loading, initiating the process in the background allows for swift commencement of new runs without unnecessary waiting period

Reply:
Done